website/content/commands/partition.mdx

```diff
@@ -134,6 +151,14 @@ $ consul partition write -format json -show-meta - <<< 'name = "webdev-bu" descr
 ### `read`
 
 The `read` subcommand sends a request to the server to read the configuration for the specified partition and print it to the console.
+This subcommand has the following characteristics:
+- **[Required ACLs](/docs/security/acl/acl-system)**: `operator:read`
+  - `none` is sufficient to allow a non-anonymous token to read its own partition
```

Comment:
I wasn't sure the best way to deal with this nuance.

The [HTTP API docs](https://www.consul.io/api-docs/admin-partitions#read-a-partition) handle this by using a footnote on `none` (see screenshot below). I'm not sure if a footnote approach like that would look strange below a list. I also wanted to see if something like this would be clearer.

![image](https://user-images.githubusercontent.com/85913323/155604457-e2ab1440-3fee-4520-b264-51179a79b548.png)

Reply:
I've tried it with and without an asterisk.